pynicotine/uploads.py

```diff
@@ -970,82 +975,72 @@ def get_upload_queue_size(self, username=None):
         if self.is_privileged(username):
             queue_size = 0
 
-            for upload in self.transfers:
-                if upload.status == "Queued" and self.is_privileged(upload.username):
+            for upload in self.queued_uploads:
+                if self.is_privileged(upload.username):
                     queue_size += 1
 
             return queue_size
 
-        return sum(1 for upload in self.transfers if upload.status == "Queued")
+        return len(self.queued_uploads)
 
     def get_downloading_users(self):
+        return set(self.active_users) + set(self.queued_users)
```

Comment:
A user may exist in both sets, but only one instance is required.

`return set(self.active_users + self.queued_users)`